schema/field/field.go

```diff
@@ -481,6 +489,14 @@ type bytesBuilder struct {
 // Default sets the default value of the field.
 func (b *bytesBuilder) Default(v []byte) *bytesBuilder {
 	b.desc.Default = v
+	b.desc.DefaultFunc = false
+	return b
+}
+
+// Default sets the default value of the field.
+func (b *bytesBuilder) DefaultFunc(v func() []byte) *bytesBuilder {
+	b.desc.Default = v
+	b.desc.DefaultFunc = true
 	return b
 }
```

Comment:
```suggestion
// Default sets the default value of the field.
func (b *bytesBuilder) DefaultFunc(v func() []byte) *bytesBuilder {
	b.desc.Default = v
	return b
}
```

Update also the godoc as mentioned in the previous comment.